src/openrct2/sprites.h

```diff
@@ -1368,7 +1368,22 @@ enum
     SPR_G2_ALPINE_LIFT_TRACK_GENTLE_DIAGONAL = SPR_G2_ALPINE_LIFT_TRACK_FLAT_DIAGONAL + 4,
     SPR_G2_ALPINE_END = SPR_G2_ALPINE_LIFT_TRACK_GENTLE_DIAGONAL + 12,
 
-    SPR_G2_ICON_PALETTE_INVISIBLE = SPR_G2_ALPINE_END,
+    SPR_G2_DIAG_BRAKES_START = SPR_G2_ALPINE_END,
```

Comment:
Make sure these are listed under     `// G2 Track additions` with the appropriate track types.